source/codegen/metadata_validation.py

```diff
@@ -218,17 +219,6 @@ def _validate_function(function_name: str, metadata: dict):
                             raise Exception(
                                 f"Failed to validate callback_param with name {callback_param['name']}"
                             )
-                if parameter.get("pointer", False):
```

Comment:
A lot of xnet inputs are passed by pointer. I could add `supports_standard_copy_convert` to this list, but there already a lot of unintuitive exceptions. Also, I don't expect someone to add this tag on accident. It might not do what they want, but this validation doesn't help explain what the gap is.